spec/requests/landing_page_spec.rb

```diff
@@ -3,6 +3,6 @@
 RSpec.describe 'Landing page', type: :request do
   it 'displays information about this project' do
     visit root_url
-    expect(page).to have_content('Project Give Back')
+    expect(page).to have_content('A project by')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.